modules/data.atmosphere/R/extract_ERA5.R

```diff
@@ -85,18 +150,33 @@ extract.nc.ERA5 <-
       all.data.point <- 
         foreach::foreach(vname = vars, 
                          .packages=c("Kendall", "ncdf4")) %dopar% {
+                           nc_data <- ncdf4::nc_open(ncfile)
+                           on.exit(ncdf4::nc_close(nc_data), add = TRUE) 
```

Comment:
will this close the connection at the end of the foreach or at the end of the overall function? If the latter, it would be easy to accidentally open too many connections and throw an error.